app/presenters/hyrax/newspaper_title_presenter.rb

```diff
@@ -5,6 +5,10 @@ class NewspaperTitlePresenter < Hyrax::WorkShowPresenter
     include NewspaperWorks::NewspaperCorePresenter
     delegate :edition, :frequency, :preceded_by, :succeeded_by, to: :solr_document
 
+    def title_search_params
+      { f: { "publication_title_ssi" => [title] } }
```

Comment:
To get this to work on my end, I had to do:
```
{ f: { "publication_title_ssi" => title } }
```
Seems the search params are literally echoing the braces around the `title`, as in `f[publication_title_ssi][]=[Mudville+Ledger]` (which doesn't match). We want it to be eval'd as `f[publication_title_ssi][]=Mudville+Ledger`.

Reply:
Sorry about that. I could've sworn this was working earlier. Should be updated now.